Add the conda packages to the documentation (#910)
metatensor/metatensor: docs/src/installation.rst

            pip install metatensor[torch]

        **Conda**

        All the packages are also available on the `conda-forge
        <https://conda-forge.org/>`_ channel, however they are named slightly
        differently. The main packages are ``metatensor`` (which installs the
        Python packages ``metatensor-core``, ``metatensor-operations`` and

>         ``metatensor-learn``); and ``metatensor-torch``, which also installs all
>         the previous packages and the ``metatensor-torch`` Python package.

Reviewer: ```suggestion
        ``metatensor-learn``); and ``metatensor-torch``, which installs all
        the previous packages and also the ``metatensor-torch`` Python package.
```

        |                                           | built.                                        |                |
        +-------------------------------------------+-----------------------------------------------+----------------+

        **Conda**

>         We also provide pre-compiled version of the C and C++ libraries on the
>         `conda-forge <https://conda-forge.org/>`_ channel. You can install it with

Reviewer: ```suggestion
        We also provide pre-compiled versions of the C and C++ libraries on the
        `conda-forge <https://conda-forge.org/>`_ channel. You can install these with
```

            listed at the end of the :ref:`install-torch-cxx` installation
            instructions.

        **Conda**

>         You can install the same code as with ``pip install metatensor[torch]``
>         using our packages in the `conda-forge <https://conda-forge.org/>`_
>         channel.

Reviewer: ```suggestion
        You can install the same code with ``pip install metatensor[torch]``
        using our packages in the `conda-forge <https://conda-forge.org/>`_
        channel.
```

          ``-DMKL_INCLUDE_DIR=/usr/include`` option.

>         **Conda**
> 
>         We also provide pre-compiled version of the library on the `conda-forge
>         <https://conda-forge.org/>`_ channel. You can install it with
> 
>         .. code-block:: bash
> 
>             conda install -c conda-forge libmetatensor-torch

Reviewer: ```suggestion
        **Conda**

        We also provide pre-compiled versions of the library on the `conda-forge
        <https://conda-forge.org/>`_ channel. You can install it with

        .. code-block:: bash

            conda install -c conda-forge libmetatensor-torch
```
